interop_tests/cli/orchestrate/operator_install.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (C) 2022 Red Hat, Inc.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+This module handles pre-defined error cases that occur in the log.
+The search result will then be saved and pass back to MPTS.
+"""
+import logging
+import os
+
+import click
+
+from interop_tests import helper
+from interop_tests.orchestrate.operator_install import OrchestrateOperatorInstall
+
+logger = logging.getLogger(__name__)
+
+
+@click.option(
+    "--operator_name",
```

Comment:
```suggestion
    "--operator-name",
```

I would suggest using dashes to separate option names.